### PR TITLE
SQL/SCRIPTS: Habetrot NM

### PR DESCRIPTION
**Pull request by dphillips2014**
Edited the Rumble Crawler that spawns next to the ??? to be the Crawler that may spawn instead of Habetrot. It's behavior is consistent with the wiki (spawns if Habetrot doesn't, can't be zoned). 

Habetrot's spawn rate is set to 20%.

### CODE REVIEW

**Comment by xdemolish** [comment]
conflicts, can't merge

File: sql/mob_spawn_mods.sql [hunk 1]
```diff
@@ -232,6 +232,8 @@ INSERT INTO `mob_spawn_mods` VALUES (16916817,49,1,1);
 
 INSERT INTO `mob_spawn_mods` VALUES (17326081,50,25,1); -- Angra_Mainyu chainspell at 25%
 
+INSERT INTO `mob_spawn_mods` VALUES (17428812,17,1,1); -- Habetrot's Rumble Crawler
```
Comment (bendangelo) [inline sql/mob_spawn_mods.sql]: No need to as `no_despawn` mob, all NMs have this by default. But I do think you should add min / max gil mods (2,000~5,625 gil)

Comment (dphillips2014) [inline sql/mob_spawn_mods.sql]: The no_despawn is for the Rumble Crawler that spawns in place of Habetrot, not Habetrot himself.

I'll add a gil mod to Habetrot.

**Comment by bendangelo** [comment]
As soon as you fix the merge conflicts this can get merged in.

**Comment by bendangelo** [comment]
A new pr was opened. https://github.com/DarkstarProject/darkstar/pull/2172

**Pull request closed without merging**